assets/models/system/microsoft-rad-dino/description.md

```diff
@@ -0,0 +1,253 @@
+
+<!--fp32, fp16 mixed precision, bf16 mixed precision, bf16 non-mixed precision, fp16 non-mixed precision, fp8 mixed precision -->
+
+## Evaluation
+
+<!-- This section describes the evaluation protocols and provides the results. -->
+
+Our evaluation is best described in the [manuscript](https://arxiv.org/abs/2401.10815).
+
+<!-- ### Testing data, factors & metrics
+
+#### Testing Data
+
+[More Information Needed]
+
+#### Factors
+
+[More Information Needed]
+
+#### Metrics
+
+[More Information Needed]
+
+### Results
+
+[More Information Needed]
+
+#### Summary -->
+
+## Environmental impact
+
+<!-- Total emissions (in grams of CO2eq) and additional considerations, such as electricity usage, go here. Edit the suggested text below accordingly -->
+
+<!-- Carbon emissions can be estimated using the [Machine Learning Impact calculator](https://mlco2.github.io/impact#compute) presented in [Lacoste et al. (2019)](https://arxiv.org/abs/1910.09700). -->
+
+<!-- Hardware type: A100 PCIe -->
+<!-- Hours: 1d 16h = 40h -->
+<!-- Cloud provider: Azure -->
+<!-- Region: Italy North -->
+
+- **Hardware type:** NVIDIA A100 GPUs
+- **Hours used:** 40 hours/GPU × 16 nodes × 4 GPUs/node = 2560 GPU-hours
+- **Cloud provider:** Azure
+- **Compute region:** West US 2
+- **Carbon emitted:** 222 kg CO₂ eq.
+
+### Compute infrastructure
+
+RAD-DINO was trained on [Azure Machine Learning](https://azure.microsoft.com/en-us/products/machine-learning).
+
+#### Hardware
+
+We used 16 `Standard_NC96ads_A100_v4` nodes with four NVIDIA A100 (80 GB) GPUs each.
+
+#### Software
+
+We leveraged the code in [DINOv2](https://openreview.net/forum?id=a68SUt6zFt) for training.
+We used [SimpleITK](https://simpleitk.org/) and [Pydicom](https://pydicom.github.io/) for processing of DICOM files.
+
+## Citation
+
+<!-- If there is a paper or blog post introducing the model, the APA and Bibtex information for that should go in this section. -->
+
+**BibTeX:**
+
+```bibtex
+@misc{perezgarcia2024raddino,
+      title={{RAD-DINO}: Exploring Scalable Medical Image Encoders Beyond Text Supervision},
+      author={Fernando Pérez-García and Harshita Sharma and Sam Bond-Taylor and Kenza Bouzid and Valentina Salvatelli and Maximilian Ilse and Shruthi Bannur and Daniel C. Castro and Anton Schwaighofer and Matthew P. Lungren and Maria Wetscherek and Noel Codella and Stephanie L. Hyland and Javier Alvarez-Valle and Ozan Oktay},
+      year={2024},
+      eprint={2401.10815},
+      archivePrefix={arXiv},
+      primaryClass={cs.CV}
+}
+```
+
+**APA:**
+
+> Pérez-García, F., Sharma, H., Bond-Taylor, S., Bouzid, K., Salvatelli, V., Ilse, M., Bannur, S., Castro, D.C., Schwaighofer, A., Lungren, M.P., Wetscherek, M.T., Codella, N., Hyland, S.L., Alvarez-Valle, J., & Oktay, O. (2024). *RAD-DINO: Exploring Scalable Medical Image Encoders Beyond Text Supervision*. ArXiv, abs/2401.10815.
+
+## Model card contact
+
+Fernando Pérez-García ([`fperezgarcia@microsoft.com`](mailto:fperezgarcia@microsoft.com)).
```

Comment:
We can remove this section. We generally do not put the contact information of one person in the model card